react-frontend/src/font-awesome/HELP-US-OUT.txt

```diff
@@ -0,0 +1,7 @@
+I hope you love Font Awesome. If you've found it useful, please do me a favor and check out my latest project,
```

Comment:
I don't think we want 'font-awesome' installed in the src of the project. Though that's more of an intuition than anything concrete.  Is it possible to import it as a package?

Reply:
removed font-awesome css/fonts